.travis.yml

```diff
@@ -26,6 +26,8 @@ before_cache:
 
 matrix:
   include:
+    - compiler: "ghc-7.4.2"
```

Comment:
please also extend the tested-with line in `cassava.cabal` so it doesn't get dropped again when regenerate the travis script